runtime/altair/src/lib.rs

```diff
@@ -1409,20 +1409,18 @@ mod upgrade {
 	impl OnRuntimeUpgrade for Upgrade {
```

Comment:
Are you ready to get this included into the runtime executive? Or is this coming in a different PR?